### PR TITLE
docs: apply the modified specs

### PR DESCRIPTION
**Pull request by audtlr24**
Add modified specs to `3-protocol-devs/1-dep-specs` and `1-users/3-data-exchange`.
- add consumer service concept
- add presentation definition

I think the presentation definition, verifiable credential, and verifiable presentation need some additional explanation page.
I will add it in next PR.

### CODE REVIEW

File: .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    - Tae Jin Yoon <tj@medibloc.org>
+
+
+## Synopsis
+
+This document describes consumer service.
+
+### Motivation
+The data consumer needs to run an HTTP server to store the provider's encrypted data.
+When creating a `deal`, the data consumer must specify the endpoint of its server in the `consumer-service-endpoint`.
```
Comment (youngjoon-lee) [inline .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md:21]:
```suggestion
Data consumers should run an HTTP server to receive the encrypted data from oracle.
When creating deals, data consumers must specify the endpoint of their server in the `consumer-service-endpoint` field.
```

File: .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+When the incentive payment to the data provider is completed, the data consumer can get the `secret key` to decrypt the stored data from the oracle. 
+Finally, the data consumer can get the original data with the data from the `consumer service` and the `secret key`.
```
Comment (youngjoon-lee) [inline .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md:24]:
```suggestion
When the incentive is paid to the data provider completely, the data consumer can get the `secret key` to decrypt data received from the oracle.
```

File: .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+#### Request Headers
+```
+# Authorization: Bearer {jwtToken}
+# Content-Type: application/json
+```
+
+The JWT is signed by oracle using the oracle private key.
+The `consumer service` can use this JWT to authenticate that oracle sent the post request.
```
Comment (youngjoon-lee) [inline .gitbook/3-protocol-devs/1-dep-specs/5-consumer-service.md:48]:
```suggestion
The `consumer service` can use this JWT to authenticate that oracle sent the POST request.
```

**Review by youngjoon-lee — APPROVED** [review]
lgtm. As MS said, we may need to put more details about VP and PD, but not urgent for now. I think it's enough to put an example about how to use VP and PD in the next PR.

**Review by gyuguen — APPROVED** [review]
LGTM

**Review by inchori — APPROVED** [review]
LGTM

**Pull request merged**